schedule/yast/btrfs/btrfs+warnings@s390x.yaml

```diff
@@ -11,35 +11,16 @@ vars:
   FILESYSTEM: btrfs
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/warning/no_root
-  - installation/partitioning/warning/snapshots_small_root
-  - installation/partitioning/warning/no_boot
-  - installation/partitioning/warning/boot_small_for_kernel
-  - installation/partitioning/warning/bios_boot_small_for_bootloader
-  - installation/partitioning/warning/prep_small
-  - installation/partitioning/warning/zipl_small
-  - installation/partitioning/warning/rootfs_small
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_ssh_service_enabled
-  - installation/installation_settings/open_ssh_port
-  - installation/installation_settings/validate_default_target
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/performing_installation/stop_timeout_system_reboot_now
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/performing_installation/reconnect_after_reboot
-  - installation/handle_reboot
-  - installation/first_boot
+  suggested_partitioning:
+    - installation/partitioning/warning/no_root
+    - installation/partitioning/warning/snapshots_small_root
+    - installation/partitioning/warning/no_boot
+    - installation/partitioning/warning/boot_small_for_kernel
+    - installation/partitioning/warning/bios_boot_small_for_bootloader
+    - installation/partitioning/warning/prep_small
+    - installation/partitioning/warning/zipl_small
+    - installation/partitioning/warning/rootfs_small
+    - installation/partitioning/accept_proposed_layout
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
+  system_preparation: []
```

Comment:
we don't need this, is already empty in the default file.

Reply:
> we don't need this, is already empty in the default file.

This case is tested with s390x kvm machine, in the kvm default file, "system_preparation" is not empty:
system_preparation:
  - console/hostname
  - console/system_prepare
  - console/force_scheduled_tasks
Then no need to update here, right? Same with the other files that I added "system_preparation: []", all using the kvm default file.

Reply:
it should be empty in the default profile, validations are always part of the test case. You can revert this problem if you want, but you can also leave it as it is. thanks for noticing it.